pkg/operator/ceph/object/objectstore.go

```diff
@@ -477,7 +477,7 @@ func createSystemUser(objContext *Context, namespace string) error {
 	zoneGroupArg := fmt.Sprintf("--rgw-zonegroup=%s", objContext.ZoneGroup)
 	zoneArg := fmt.Sprintf("--rgw-zone=%s", objContext.Zone)
 
-	output, err := RunAdminCommandNoMultisite(objContext, false, "user", "info", uidArg)
+	output, err := RunAdminCommandNoMultisite(objContext, false, "user", "info", uidArg, realmArg)
```

Comment:
I thought realm, zone and zonegroup are always added to the `radosgw-admin` commands. Anyways it will be if you include zone and zoneGrouparg in my opinion

Reply:
Should this be calling `runAdminCommand()` instead of `RunAdminCommandNoMultisite()`? By calling the latter directly, looks like the realm/zone/zonegroup args are being skipped.

Reply:
Yaa that should alsoe work IMO may be same lineno 495 as well, we can also remove 474-478 in that case

Reply:
It may be that this is `RunAdminCommandNoMultisite()` in this context because the RGWs for the object store haven't been created yet by this point in the process. I think it may be most correct to keep using `RunAdminCommandNoMultisite()`

Reply:
I made the requested change